.travis.yml

```diff
@@ -28,7 +28,7 @@ notifications:
 deploy:
   provider: heroku
   api_key:
-    secure: iXCd16jR5WF25XMDNDhDx8b2JC+UIl/l3dtqN2kV/a1PSYjgFQB0VfcMih+mvAdOlVyeVMK2wckmDmc4xoxeqP2Ki9o6P5kz/Wdqgfohv4gIQZnfvAa0VmFTob4ckCRlJF/cF/x8DPiKrdpTreNI3g2VjUIdCtX6zXnHakwpswhYC4yKRy5KfrCKAFLqp2H6LtQJIlx8t//yXqQQA1Lpo1pEj2HIteuUq+nYfeBdHYWlTgczz++7048YbCPZiOyjEcLd1RGYyU2ddMiSbSD9fidiqBTJMpdIpwXztnqL2JMwPiFH0WfisDgo6wFMi6iA3sud4/eqY72nb09BWmDQvwxbG0baGF/svwFfGv99OeH8O4UgsOFYgFSdp/Ksg3Xqbr/7IszIWxcL5+oRiQibtS2hZ9fzhM6iIz3RekAF2/m9XS0Q4QzEL9Z5Jp6hNH0SUe7w7ui1yF1iubjlh0QNvXmChe7O5prO0vAMq+PFI+DxJmBu5l2r3It/X8H9kKja4DrRWMHR2ImFM72kqh9BOYmtR4nQeAllddNu095k8BOg2Md5p7T4A4kvAztjLyNr4iFNfc1g4umegOSZyFwhfk9rt54VS4jLr9MwMC5BX0e9jInXcvjGP89e106S8OCF1xoVv7LgHbNg+OX0ktn3p39SyrikFavwyeP9DJTvPPo=
+    secure: LSBCTDe94MVR8V7usyQBstsbd165GN6Gmh6/CAc5I61oVT1RPBY2hKeacT0rwfCS9y4pIpXrUBQyMZbA7nEUtOulsQJ4i8T0Oin5dqSchJAhWJOdOtgNE2EQrbKVc4Xoz2AfA0uci7/icAf4mEfqFi0MPB8qBEkvxsq9FyceMuINdsYRF+hWbsgyr42xUtuTPLBQnQP+DD+hhkyQQXUZTxOXa7VUQ0A8X8EEZ6oZigKLApMf1cwafzf64vaTxX5IXv03OWDPe0f1ekNdULFPCEcQSc2aeSRsNnVXMtM8v0e+ZZ3NzHswgp8+g15l75v5FtbUXTVo4510Br6DKkRjoLRCqf/1XltlWwfBwkHGMOzE3+9XPvtsu5NhbY9qEese+E/x+khz6mZh5U223YVpHfhaeq3NfoTs9wLfMaq+FCZv3tKFWVDeBH6fHbDS5h/mdequS8GddsuoA8kWQPzOzEo1KOf7QZ6C5cXAFjJRDkUWvduUeCFaUgcxDiRvTR0aJecyi8hrZ24UHdwP49Y+tjCImiBpSDYozEDTEtEQlS/TyYfPGGgj1ntGg1cfsIvhRkDje46WIa3kqkbsm8xfJF8Bk6T8F583GcONOLxh/Q7po6nROsnUvt9BJDDLVLbA+zDqCJa+nthBjS6IpFV87gmbbwjOItoGun5bMvID15s=
```

Comment:
Great work @devdbrandy,
Would it be possible to add the heroku API key as an environment variable on Travis CI? You could then reference that variable on this .travis.yml file ($HEROKU_API_KEY). That way, if we have any reason to regenerate the Heroku API key, we would not need to update this file, commit and push changes to github. Instead, we could just update the key on Travis CI only and everything works smoothly.

Let me know if this is something you can do.

Reply:
Hi @valenstical, you have made very good point. However, its rather too early in travisCI's sequence for environment variable to be read, which is why travis has provided the ability for us to encrypt sensitive data such as api credentials.

Reply:
Oh, that sounds good. Thanks for clarifying that.